atom/renderer/lib/web-view/web-view.js

```diff
@@ -391,14 +391,16 @@ var registerWebViewElement = function() {
     'printToPDF'
   ];
   nonblockMethods = [
-    'executeJavaScript',
     'insertCSS',
-    'insertText',
     'send',
-    'sendInputEvent',
```

Comment:
Trailing commas should be kept in lists, see:
http://www.puppetcookbook.com/posts/trailing-commas.html